### ISSUE TITLE
Server Not Showing Up In ClassiCube Server List Only Showing Up On The Web Version

### ISSUE THREAD

**Issue by RafCodes-del** [ClassiCube/MCGalaxy#698]
I Hosted A Server That Uses This Server Hosting Software I Hosted It I Allowed Firewall But If I Search My Server On The Server List It Doesn't Show Up I Tried Joining By Link I Copied The Link Still No Please Fix This

**Comment by forkiesassds** [comment]
Did you set public to true in server.properties?

**Comment by RafCodes-del** [comment]
Yes, I Set The Server.properties To True But Still Doesen't Show Up I May Have Checked My Firewall Settings Its Allowed Still

**Comment by UnknownShadow200** [comment]
Are you seeing a message like ` {"errors":[["Port 25565 not open, you may need to port forward it."]],"response":"","status":"fail"}
` in the server logs?

**Comment by RafCodes-del** [comment]
Yes. I did see a message like that.

**Comment by UnknownShadow200** [comment]
Then most likely you will need to port forward in your network router

Unfortunately, how exactly you port forward is router specific - portforward.com does provide guides for a lot of different routers, but make sure not to download any software from there
